acceptance-tests/features/steps/borrower_views_deed_definitions.rb

```diff
@@ -0,0 +1,48 @@
+Given(/^I navigate to the deed search page$/) do
+  visit("#{$CHARGES_URL}/deed/search")
+end
+
+When(/^I search for a deed with reference "([^"]*)"$/) do |arg1|
+  fill_in 'deedRefNumber', with: arg1
+  click_button('Search')
+end
+
+Then(/^the Sign your mortgage deed page is displayed$/) do
+  assert_selector(".//*[@id='content']/div/h1", text: 'Sign your mortgage deed')
```

Comment:
I would recommend not using xpath at all.

The html should correct label a link, one due to accessibility but also to make testing easier.